### PR TITLE
[CHORE] Update tide-disco (and others) to crates.io

### PR DESCRIPTION
**Pull request by jbearer**

### CODE REVIEW

**Review by ggutoski — COMMENTED** [review]
jellyfish update and `vid.rs` LGTM but I'll let others approve the rest.

**Review by ss-es — DISMISSED** [review]
Looks straightforward, and I trust Gus on the `vid.rs` change

**Review by jparr721 — APPROVED** [review]

**Review by rob-maron — APPROVED** [review]

File: Cargo.toml [hunk 1]
```diff
@@ -35,7 +35,7 @@ ark-ff = "0.4"
 ark-serialize = "0.4"
 ark-std = { version = "0.4", default-features = false }
 async-broadcast = "0.7"
-async-compatibility-layer = { git = "https://github.com/EspressoSystems/async-compatibility-layer.git", tag = "1.4.2", default-features = false, features = [
+async-compatibility-layer = { version = "1.1", default-features = false, features = [
```
Comment (nyospe) [inline Cargo.toml:38]: 👍

File: Cargo.toml [hunk 2]
```diff
@@ -49,7 +49,7 @@ bitvec = { version = "1", default-features = false, features = [
   "serde",
 ] }
 blake3 = "1.5"
-commit = { git = "https://github.com/EspressoSystems/commit" }
+committable = "0.2"
```
Comment (nyospe) [inline Cargo.toml:52]: 👍

**Review by nyospe — DISMISSED** [review]
LGTM

**Review by rob-maron — APPROVED** [review]

**Pull request merged**